features/step_definitions/events.js

```diff
@@ -21,6 +23,65 @@ Given(/^an existing pipeline with the workflow:$/, { timeout: TIMEOUT }, functio
 
 Given(/^"calvin" has admin permission to the pipeline$/, () => null);
 
+Given(/^the "main" job has a previous event$/, { timeout: TIMEOUT }, function step() {
+    const jobName = 'main';
+    return request({
+        url: `${this.instance}/${this.namespace}/events`,
+        method: 'POST',
+        json: {
+            pipelineId: this.pipelineId,
+            startFrom: jobName
+        },
+        context: {
+            token: this.jwt
+        }
+    })
+        .then(resp => {
+            Assert.equal(resp.statusCode, 201);
+            this.previousEventId = resp.body.id;
+        })
+        .then(() =>
+            sdapi.cleanupBuilds({
```

Comment:
No need to wait for the build of the parent event to succeed, so we can abort it.